pwrsupply/bsmp-dclink.txt

```diff
@@ -221,10 +221,10 @@ SI-Fam:PS-QDP1     PA-RaPSA03:PS-DCLink-QDAP
 SI-Fam:PS-QDP2     PA-RaPSA03:PS-DCLink-QDAP
 SI-Fam:PS-QDB1     PA-RaPSA04:PS-DCLink-QDB
 SI-Fam:PS-QDB2     PA-RaPSA04:PS-DCLink-QDB
-SI-Fam:PS-Q1       PA-RaPSA06:PS-DCLink-Q12A PA-RaPSA06:PS-DCLink-Q12B PA-RaPSA06:PS-DCLink-Q12C
-SI-Fam:PS-Q2       PA-RaPSA06:PS-DCLink-Q12A PA-RaPSA06:PS-DCLink-Q12B PA-RaPSA06:PS-DCLink-Q12C
-SI-Fam:PS-Q3       PA-RaPSA07:PS-DCLink-Q34A PA-RaPSA07:PS-DCLink-Q34B PA-RaPSA07:PS-DCLink-Q34C
-SI-Fam:PS-Q4       PA-RaPSA07:PS-DCLink-Q34A PA-RaPSA07:PS-DCLink-Q34B PA-RaPSA07:PS-DCLink-Q34C
+SI-Fam:PS-Q1       PA-RaPSA06:PS-DCLink-Q13A PA-RaPSA06:PS-DCLink-Q13B PA-RaPSA06:PS-DCLink-Q13C
+SI-Fam:PS-Q2       PA-RaPSA06:PS-DCLink-Q24A PA-RaPSA06:PS-DCLink-Q24B PA-RaPSA06:PS-DCLink-Q24C
+SI-Fam:PS-Q3       PA-RaPSA07:PS-DCLink-Q13A PA-RaPSA07:PS-DCLink-Q13B PA-RaPSA07:PS-DCLink-Q13C
+SI-Fam:PS-Q4       PA-RaPSA07:PS-DCLink-Q24A PA-RaPSA07:PS-DCLink-Q24B PA-RaPSA07:PS-DCLink-Q24C
```

Comment:
There is an inconsistency here with the subsections: 
PA- **RaPSA06** :PS-DCLink-Q13A and PA- **RaPSA07** :PS-DCLink-Q13A